modules/serverless-rn-1-20-0.adoc

```diff
@@ -0,0 +1,146 @@
++
+For example, an event delivery to a publicly accessible address works:
++
+[source,terminal]
+----
+$ kn event send --to-url https://ce-api.foo.example.com/
+----
++
+On the other hand, this delivery fails if the service uses a public address with an HTTPS certificate issued by a custom CA:
++
+[source,terminal]
+----
+$ kn event send --to Service:serving.knative.dev/v1:event-display
+----
++
+Sending an event to other addressable objects, such as brokers or channels, is not affected by this issue and works as expected.
+
+* The Kafka broker currently does not work on a cluster with Federal Information Processing Standards (FIPS) mode enabled.
+
+* If you create a Springboot function project directory with the `kn func create` command, subsequent running of the `kn func build` command fails with this error message:
++
+[source,terminal]
+----
+[analyzer] no stack metadata found at path ''
+[analyzer] ERROR: failed to : set API for buildpack 'paketo-buildpacks/ca-certificates@3.0.2': buildpack API version '0.7' is incompatible with the lifecycle
+----
++
+As a workaround, you can change the `builder` property to `gcr.io/paketo-buildpacks/builder:base` in the function configuration file `func.yaml`.
+
+* Deploying a function using the `gcr.io` registry fails with this error message:
++
+[source,terminal]
+----
+Error: failed to get credentials: failed to verify credentials: status code: 404
+----
++
+As a workaround, use a different registry than `gcr.io`, such as `quay.io` or `docker.io`.
+
+* Functions created with the TypeScript runtime and the `http` template do not get deployed on the cluster.
++
+This issue can be reproduced with these steps:
++
+[source,terminal]
+----
+$ kn func create -l typescript -t http /tmp/ts-http
+Created typescript Function in /tmp/ts-http
+----
++
+[source,terminal]
+----
+$ kn func deploy -r docker.io/jrangelramos -p /tmp/ts-http
+   🙌 Function image built: docker.io/jrangelramos/ts-http:latest
+🕖 Deploying function to the cluster
+Error: knative deployer failed to wait for the Knative Service to become ready: timeout: service 'ts-http' not ready after 120 seconds
+----
++
+[source,terminal]
+----
+$ kn func run -p /tmp/ts-http
+cat: /sys/fs/cgroup/memory/memory.limit_in_bytes: No such file or directory
+/layers/paketo-buildpacks_node-engine/node/profile.d/0_memory_available.sh: line 3: / ( 1024 * 1024 ) : syntax error: operand expected (error token is "/ ( 1024 * 1024 ) ")
+Error: Cannot find module '/workspace/build/index.js'
+Require stack:
+
+    /layers/paketo-buildpacks_npm-install/modules/node_modules/faas-js-runtime/bin/cli.js
+----
++
+As a workaround, in the `func.yaml` file, replace the following section:
++
+[source,terminal]
+----
+buildEnvs: []
+----
++
+with this:
++
+[source,terminal]
+----
+buildEnvs:
+- name: BP_NODE_RUN_SCRIPTS
+  value: build
+----
++
+TypeScript functions with the `event` templates do not have this issue.
```

Comment:
Here a suggestion... I think I would probably omit this line as well.. once there is no issue with cloudevents template for Typescript. That was just a side note for devs...
But in case you leave this line use `cloudevents` instead of `event`.

Reply:
Omitted the line.